src/main/kotlin/no/nav/familie/ef/sak/vurdering/medlemskap/Medlemskapshistorikk.kt

```diff
@@ -101,8 +101,8 @@ class Medlemskapshistorikk(pdlPerson: PdlPerson, medlemskapsinfo: Medlemskapsinf
 
     private fun mapTilBosattperioder(bostedsadresser: List<Bostedsadresse>): List<Periode> {
         val bosattAdresser = bostedsadresser
-                .filter { it.angittFlyttedato != null || it.folkeregistermetadata.gyldighetstidspunkt != null }
-                .sortedBy { it.angittFlyttedato ?: it.folkeregistermetadata.gyldighetstidspunkt!!.toLocalDate() }
+                .filter { it.gyldigFraOgMed != null }
+                .sortedBy { it.gyldigFraOgMed ?: it.gyldigFraOgMed }
 
```

Comment:
Denne er jeg litt usikker på. Det er mulig vi burde hente angittFlyttedato og bruke den som fallback?
Se slacktråden som linkes til

Reply:
Skulle gjerne verifisert hvor "bra" gyldigFraOgMed er - hvis denne er god nok for vår del så kunne vi bare holdt oss til den, men hvis ikke så trenger vi nok angittFlytteDato både her og i DTO-en tror jeg...